src/components/Event/Event.tsx

```diff
@@ -1,72 +1,37 @@
-import React from 'react';
-import styled from 'styled-components';
+import { FC } from 'react';
 import Button from '../Button/Button';
+import Description from '../Description/Description';
 import Title from '../Title/Title';
+import { EventWrapper, DateWrapper, DescriptionWrapper, ColoredYear } from './Event.styles';
 
-interface EventProps {
-  date: Date;
-  description: string;
-  background?: string;
-  title: string;
+export interface EventProps {
+    description: string;
+    background?: string;
+    title: string;
+    eventDate: Date;
 }
 
-interface EventWrapperProps {
-  background?: string;
-}
-
-const EventWrapper = styled.div<EventWrapperProps>`
-  background-color: ${({ theme, background }) => (background ? background : theme.backgrounds.transparent)};
-  color: ${({ theme }) => theme.colors.white};
-  border: 0.5px solid ${({ theme }) => theme.colors.white};
-  padding: 20px;
-  max-width: 290px;
-  height: 290px;
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  flex-direction: column;
-`;
-
-const DateWrapper = styled.div`
-  display: flex;
-  justify-content: space-between;
-  width: 100%;
-  letter-spacing: 1px;
-  font-size: ${({ theme }) => theme.fontSizes.s};
-`;
-
-const DescriptionWrapper = styled.div`
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  width: 100%;
-  max-height: 50%;
-  font-size: ${({ theme }) => theme.fontSizes.m};
-  font-weight: ${({ theme }) => theme.fontWeights.thin};
-`;
-
-
-
-const ColoredYear = styled.span`
-  color: ${({ theme }) => theme.colors.rose};
-`;
-
-const Event = ({title, description, background }: EventProps) => {
-
-  return (
-    <EventWrapper background={background}>
-      <DateWrapper>
-        <div>
-          <ColoredYear>2021</ColoredYear>{' / 02 / 03'}
-        </div>
-        <div>20:41:06 GMT</div>
-      </DateWrapper>
-      <DescriptionWrapper>
-        <Title title={title} />
-        <div>{description}</div>
-      </DescriptionWrapper>
-      <Button/>
-    </EventWrapper>
-  );
+const Event: FC<EventProps> = ({ title, description, background, eventDate }) => {
+    const year = eventDate.getFullYear();
+    const month = eventDate.getMonth() + 1;
+    const day = eventDate.getDate();
+    console.log(eventDate);
+
+    return (
+        <EventWrapper background={background}>
+            <DateWrapper>
+                <div>
+                    <ColoredYear>{year}</ColoredYear>
+                    {` / ${month < 10 ? `0${month}` : month} / ${day < 10 ? `0${day}` : day}`}
```

Comment:
umm, quite unusual conditions. Consider make iso string from date object and parse it accordingly, e.g:

```suggestion
  const year = date.getFullYear();
  const month = date.toLocaleDateString().split('/')[1];
  const day = date.toLocaleDateString().split('/')[0];
  const time = date.toLocaleTimeString();
```

Reply:
created new const + comment